cluster/manifests/skipper/deployment.yaml

```diff
@@ -256,6 +256,7 @@ spec:
           - "-disabled-filters={{ .Cluster.ConfigItems.skipper_disabled_filters }}"
 {{ if ne .Cluster.ConfigItems.skipper_routesrv_enabled "exec" }}
           - '-default-filters-prepend={{ .Cluster.ConfigItems.skipper_default_filters }}'
+          - '-default-filters-append={{ .Cluster.ConfigItems.skipper_default_filters_authentication }}'
```

Comment:
Its OK to use empty flag value, see https://github.com/zalando/skipper/pull/3084